packages/foreman/foreman/foreman-sidekiq

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /usr/bin/scl enable tfm "sidekiq $*"
```

Comment:
Rather than this wrapper, can't we use `/usr/bin/tfm-ruby` which basically already does this? Can we call `/opt/theforeman/tfm/root/usr/bin/sidekiq` and put that in the right context?

Reply:
We cannot, unfortunately. This would mean every Ruby application would end up in `foreman_rails_t` domain, which we don't want to.

Your second suggestion is possible, however again this means every single sidekiq instance would run in our Foreman RoR domain which might not be wanted.

The wrapper is the cleanest solution, I used `exec` so there is not another subprocess hanging in the `ps` output.

Reply:
> We cannot, unfortunately. This would mean every Ruby application would end up in `foreman_rails_t` domain, which we don't want to.

I meant that a better wrapper is doing something like (untested):
```suggestion
exec /usr/bin/tfm-ruby "sidekiq $*"
```

The benefit is that we can use this on both EL7 and EL8 since `/usr/bin/tfm-ruby` is a constant on all platforms. Not sure it will find sidekiq though.